src/templates/drizzle/page-item.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="drizzle-Item">
+  <div class="drizzle-Grid drizzle-Grid--withGutter">
+    <div class="drizzle-Grid-cell drizzle-u-md-size1of3">
+      <div class="drizzle-FrameThumb" data-drizzle-append-iframe="./{{@key}}.html" aria-hidden="true">
+        <a href="./{{@key}}.html"></a>
+      </div>
+    </div>
+    <div class="drizzle-Grid-cell drizzle-u-md-size2of3">
+      <div class="drizzle-u-marginEnds">
```

Comment:
This reminds me of something that slipped through...

Since this piece of UI seems to occur often, we should have a partial for it: https://github.com/cloudfour/cloudfour.com-patterns/blob/master/src/templates/drizzle/labelheader.hbs